src/components/buttonsIndicators/buttonSecondary/index.tsx

```diff
@@ -15,8 +15,8 @@ export interface Props {
   icon?: {image: React.ReactNode, position: 'left' | 'right'}
 }
 
-export type Color = 'brand' | 'action' | 'subtle'
-export type Size = 'large' | 'medium' | 'small'
+export type Color = 'brand' | 'action' | 'subtle' | 'donate'
+export type Size = 'full' | 'large' | 'medium' | 'small'
```

Comment:
Please don't add new sizes that are not defined in design language. Same as above